### ISSUE TITLE
Comment to a milestone

### ISSUE THREAD

**Issue by chris-b-pacon** [pinosante/VAM-Evolutionary-Character-Creation#42]
I do not know how to comment on the goals of a milestone, so I try it this way:

Why is it important to have the ratings working in parallel? Why not just optimize head first, and the switch to body? I imagine it is viable to switch morphs groups to evaluate for each new generation.

**Comment by pinosante** [comment]
You are right. On the backend, you could do that. Since the 500 morphs you have, are split into a body and head group which are exclusive, you could first do the run for the heads and then do another the selection run for the body and only overwrite the morph values for the body morphs. There is one thing I didn't consider, and that is that it kind of stops being a "pure" genetic algorithm, since you stop using strictly two parents to create 1 child, but instead have two parents to create the head from, and another two parents to create the body from. But as an algorithm it can definitely work.
